docs/how-to-guides/extended-query-tags.md

```diff
@@ -74,17 +74,17 @@ Add extended query tags to supported set with route:  /extendedquerytags
 		"Path":"04011001",
 		"VR":"SS",
 		"PrivateCreator":"MicrosoftPC",
-		"Level":"Instance"
+		"QueryTagLevel":"Instance"
```

Comment:
if we use _QueryTagLevel_ here, should we also append _QueryTag_ to other properties?

Reply:
QueryTagPath, QueryTagVR etc.